handlers/main.yml

```diff
@@ -1,10 +1,8 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put handlers for tasks here.
+- name: Write boot loader configuration
+  command: grub2-mkconfig -o {{ __bootloader_grub_conf }}
+  changed_when: true
 
-# Example:
-- name: Bootloader handler to restart services
-  service:
-    name: "{{ item }}"
-    state: restarted
-  loop: "{{ __bootloader_services }}"
+- name: Reboot system
+  include_tasks: handlers/reboot.yml
```

Comment:
I would prefer to have this under `tasks/`

Reply:
Agreed, moved